.github/workflows/nightly-builds.yml

```diff
@@ -2,7 +2,7 @@ name: Nightly builds
 
 on:
   schedule:
-    - cron: '0 0 * * *' # Run at 12 AM UTC.
+    - cron: '1 0 * * *' # Run at 12:01 AM UTC.
```

Comment:
The date was changed here to run at 1 minute past midnight in order for the date calculation to be accurate (as I don't _think_ it takes time into account. We subtract 1 day from the current day, which is ambiguous at 00:00.